catroid/src/main/java/org/catrobat/catroid/ui/dragndrop/BrickListView.kt

```diff
@@ -0,0 +1,282 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
```

Comment:
update copyright, will not be changed because it is a new file

Reply:
@Walcho1125 Thanks, done!